x/leveragelp/keeper/begin_blocker.go

```diff
@@ -106,19 +119,19 @@ func (k Keeper) ClosePositionIfUnderStopLossPrice(ctx sdk.Context, position *typ
 	h, err := k.GetPositionHealth(ctx, *position)
 	if err != nil {
 		ctx.Logger().Error(errors.Wrap(err, fmt.Sprintf("error updating position health: %s", position.String())).Error())
-		return false, true, err
+		return false, false, err
 	}
 	position.PositionHealth = h
 	k.SetPosition(ctx, position)
 
 	lpTokenPrice, err := ammPool.LpTokenPrice(ctx, k.oracleKeeper)
 	if err != nil {
-		return false, true, err
+		return false, false, err
```

Comment:
used in close_positions

Reply:
Checking there through error. If this function doesn't close the position, it always return an error